SECP-service/src/main/resources/assets/app/js/profiles/group-profile-controller.js

```diff
@@ -0,0 +1,3 @@
+angular.module('SECP')
+    .controller('GroupProfileController', ['$scope', function ($scope) {
```

Comment:
We'll update this logic later eh?

Reply:
it doesnt really need one I dont think. All it should do for the group profiles is show the group info. Only admins should be able to edit a group if needed.